synced-files/.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,51 @@
+name: Feature Request
+description: Request a new feature or enhancement
+labels: [T-Enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to suggest improvements to Realm!
+  - type: textarea
+    id: problem
+    attributes:
+      label: Problem
+      description: A clear and concise description of the problem you are trying to solve.
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: Solution
+      description: Describe the solution you envision, including API and usage example if possible.
+    validations:
+      required: false
+  - type: textarea
+    id: alternative-solution
+    attributes:
+      label: Alternatives
+      description: Describe the alternative solutions or features you have considered
+    validations:
+      required: false
+  - type: dropdown
+    id: importance
+    attributes:
+      label: How important is this improvement for you?
+      options:
+        - Choose!
```

Comment:
This is important so we "force" people to select.

Reply:
Can't find any docs about this - is `Choose!` somehow special cased by the form renderer? Because it looks like just another option here, meaning people would be able to select "Choose!" as an answer, which is obviously not what we're trying to achieve.

Reply:
I updated to "-- select --", but besides the name-change it's because there is a bug in forms now that means that even when you add "validation:true" nothing is really forcing you to make a choice and therefore I'm afraid we end up with people choosing the first option and it's wrong. If you add "validation:false" an automatic default option "None" is actually added, but without the validation to force you to choose.. :-(

If they choose "-- select --" - i.e. don't make a choice, at least we can see that now.
And potentially post an auto response to choose better responses.